application/Profile/DBProfile.py

```
@@ -97,16 +114,27 @@ def validate_username(self, key, username):
 
             @validates('email')
             def validate_email(self, key, email):
-                assert re.match(r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b', email), \
-                    f'Invalid email address: "{email}"'
+                invalid_email_error = f'Invalid email address: "{email}"'
+                try:
+                    assert re.match(r'\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b', email)
```

Ruff

Use of `assert` detected

(S101)